### ISSUE TITLE
Crash in PushServiceApi26

### ISSUE THREAD

**Issue by iernie** [parse-community/Parse-SDK-Android#744]
Hi

I know you guys have been working on this service lately and wanted to let you know that I am experiencing a crash when getting push on API 26.

The push notification did arrive on my phone, I did see it and could click it. I couldn't see any error from my phone on that regard, but I did get this message in Fabric immediately after.

```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke interface method 'void com.parse.PushHandler.handlePush(android.content.Intent)' on a null object reference
       at com.parse.PushServiceApi26$1.run(PushServiceApi26.java:92)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
       at java.lang.Thread.run(Thread.java:764)
```

Good thing is that you don't notice this crash as a user, but would be nice to see what the issue is. SDK 1.16.2.

**Comment by natario1** [comment]
This is confusing. Maybe onCreate() is not called at the right time for `JobService`s and we only manage to deliver the notification when it is rescheduled. But does not make much sense to me

**Comment by steven274** [comment]
I have the same problem, please help

**Comment by rogerhu** [comment]
Released on 1.16.3

**Comment by meshileya** [comment]
I encountered the same problem not quite long.
